backend/lib/routes/shoots.js

```diff
@@ -254,3 +254,17 @@ router.route('/:name/spec/seedName')
       next(err)
     }
   })
+
+router.route('/:name/adminkubeconfig')
+  .all(metricsMiddleware)
+  .post(async (req, res, next) => {
+    try {
+      const user = req.user
+      const namespace = req.params.namespace
+      const name = req.params.name
+      const expirationSeconds = req.body.expirationSeconds
+      res.send(await shoots.createAdminKubeconfig({ user, namespace, name, expirationSeconds }))
```

Comment:
pass the complete body as body to the `createAdminKubeconfig` service function.

Reply:
Changed it